app/models/vm_migrate_workflow.rb

```diff
@@ -38,6 +40,15 @@ def get_source_and_targets(refresh = false)
     @target_resource = result
   end
 
+  def add_target(dialog_key, key, klass, result)
+    super unless field_unsupported(key)
```

Comment:
It would be great if logic was positive case rather than the current double negative "unless unsupported"